apps/chanakya/core/chat_service.py

```
@@ -1592,12 +1603,37 @@ def _chat_internal(
+                img_path = img_dir / img_filename
+                try:
+                    with open(img_path, "wb") as f:
+                        f.write(base64.b64decode(raw_b64))
+                    image_files = metadata.get("image_files", [])
+                    image_files.append({
+                        "path": str(img_path.relative_to(get_data_dir())),
+                        "media_type": media_type,
+                    })
+                    metadata["image_files"] = image_files
+                except Exception:
+                    pass
```

Ruff (0.15.12)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)